### DISCUSSION TITLE
how to using flow.visualize() on window 11

### DISCUSSION THREAD

**Discussion by nguyenngothuong** [PrefectHQ/prefect#10616] [Q&A]
Hi guys, I'm using prefect on my personal laptop, I want to use the "visualize" feature to see how my data path works.
I installed the included graphviz library but it doesn't seem to work on my machine.
Thank you for reading

**Comment by discdiver** [comment]
Thank you for your message. Do you have an error traceback you can share? [Here's a Stack Overflow discussion](https://stackoverflow.com/questions/72754723/graphviz-installation-and-the-instruction-for-use) with info on graphviz installation in case there were any issues there. From an answer in that thread, it looks like VSCode also has an extension if you're using that.
